pkg/token/server.go

```diff
@@ -78,7 +78,8 @@ func postRoleToken(ts *tokenService, w http.ResponseWriter, r *http.Request) {
 	}
 
 	// create cache key
-	k := CacheKey{Domain: domain, Role: role}
+	// Prioritize checking the cases that are subject to file output.
+	k := CacheKey{Domain: domain, Role: role, WriteFileRequired: true}
```

Comment:
@mlajkim 
Check if this should be true

Reply:
logically np in the current latest commit.
Yet for better readability for humans, its better to set it `false` and do something else

Reply:
fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/131/commits/c2df53383b2bc00be4658a1416cae5b696b4697b